### ISSUE TITLE
[Records/Dictionaries] Expand `on:` in attachments

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#1011]

**Comment by jstanden** [comment]
* [Records/Attachments] In attachment dictionaries, the `on` expansion key returns information about the records the file is linked to. By default `expand:[on]` returns an array of linked record dictionaries with `_context` and `id` keys. The returned record types can be filtered by specifying them like `expand:[on.comment,on.message]`, which returns only information about comments and messages each attachment is linked to (ignoring other types). The returned record dictionaries also support key expansion like `expand:["on.message:_label,custom_,ticket_group_"]`. Multiple expansion keys are separated with commas. Note that the expansion pattern is double-quoted. 

~~~
type:worklist.records
of:attachment
query:(name:*.png on:comment)
expand:["on.comment:_label,author_,custom_"]
format:dictionaries
~~~

**Comment by jstanden** [comment]
Implemented in 9.4
